python/ground_git.py

```diff
@@ -0,0 +1,1056 @@
+# /usr/bin/env python3
+import requests
+import json
+import numpy as np
+import os
+import git
+import subprocess
+from shutil import copyfile
+# noinspection PyUnresolvedReferences
+from ground.common.model.core.node import Node
+# noinspection PyUnresolvedReferences
```

Comment:
What are these comments?

Reply:
Those are just for my IDE, PyCharm, so it doesn't stop a run for errors. For whatever reason, PyCharm doesn't recognize the importing of local packages, so it will show an error that is not really there. I can remove those if that is preferable

Reply:
Yeah, please remove them from the code.

Reply:
Bump on this.